packages/site/src/views/homepage/index.vue

```diff
@@ -10,7 +10,7 @@ function goToUse() {
 <template>
   <div class="home-container">
     <a href="https://github.com/sword-x-shield/pizza-ui" target="_blank">
-      <img src="../../assets/pizza.png" alt="Pizza logo">
+      <img src="../../assets/pizza-background.png" alt="Pizza logo">
     </a>
     <h1>Pizza UI</h1>
     <h2>It's a useless ui library!!!</h2>
```

Comment:
Based on the given code patch, it seems like a simple change where the "alt" attribute of the "img" tag has been updated to "Pizza logo" from "Pizza logo" which is already specified.

There is not enough context available to say anything else regarding possible bug risks or improvement suggestions.